src/main/scala/com/wavesplatform/general/BlockVersionResolver.scala

```diff
@@ -0,0 +1,12 @@
+package com.wavesplatform.general
```

Comment:
This is overengineered, we only need 1-line function, so make it function and call where needed